Eastern-Research-Group/expert-query
# Feature/133 input values endpoint

## Related Issues:
* [EQ-133](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=865&view=detail&selectedIssue=EQ-133)

## Main Changes:
* Added a `values` endpoint to the internal API, used for populating inputs in the interface. This endpoint will return column values that match a case-insensitive `LIKE` clause up to a specified limit, and optionally filter the results by other `WHERE` clauses.
* Updated the select inputs in the interface to send a request to the endpoint if field values are not already present in the domain values or list values.

## Steps To Test:
1. Start the client application, then navigate to the home page: https://localhost:3000/attains.
2. In the **Data Profile** drop-down, select *Assessment Units*.
3. The **Assessment Unit ID** field and the **Location Text** field both retrieve their values from the database. Confirm that they are populated with 20 values by default, and that the list of values updates as characters are entered into the input's search box.
4. Confirm that the **Location Text** field values update properly when different sources are selected from the source drop-down.
